blocks.md

```diff
@@ -378,8 +378,12 @@ step_width = 3
 
 Key | Values | Required | Default
 ----|--------|----------|--------
-`step_width` | The percent volume level is increased/decreased for the selected audio device when scrolling. Capped automatically at 50. | No | `5`
-`on_click` | Shell command to run when the sound block is clicked. | No | None
+path | Path to collect information from | No | /
```

Comment:
Uhm, this is does not correspond to the Sound block. Maybe you edited the wrong lines?

Reply:
This file just keeps messing itself up for some reason.. trying to solve it again.